frontend/javascripts/oxalis/view/layouting/tracing_layout_view.js

```diff
@@ -343,6 +349,8 @@ function mapStateToProps(state: OxalisState): StateProps {
     storedLayouts: state.uiInformation.storedLayouts,
     isDatasetOnScratchVolume: state.dataset.dataStore.isScratch,
     datasetName: state.dataset.name,
+    displayName: state.tracing.name ? state.tracing.name : state.dataset.name,
+    organization: state.activeUser ? state.activeUser.organization : "",
```

Comment:
Does this mean that no organization is shown if the user is not logged in? I think, this is problematic since the organization title should especially be presented for guest users.

Reply:
Should it be the organization that owns the dataset instead?

Reply:
Yes, I think that makes sense!